clang/lib/Driver/Driver.cpp

```diff
@@ -1326,6 +1326,20 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
       }
     }
   }
+  // -fsycl-fp64-conv-emu is valid only for AOT compilation with an Intel GPU
+  // target. For other scenarios, we emit a warning message.
+  if (C.getInputArgs().hasArg(options::OPT_fsycl_fp64_conv_emu)) {
+    bool HasIntelGPUAOTTarget = false;
+    for (auto &TT : UniqueSYCLTriplesVec) {
+      if (TT.isSPIRAOT() && TT.getSubArch() == llvm::Triple::SPIRSubArch_gen) {
```

Comment:
NIT: `LLVM coding standards` recommend descriptive variable names.

Reply:
I suppose you are referring to TT here. I looked at Driver.cpp and this variable is used in several locations and seems agreeable. I would suggest a separate pass to rename all these uses. Thanks